src/main/java/com/teamacronymcoders/essence/items/tools/EssenceAxe.java

```diff
@@ -52,37 +52,37 @@ public void fillItemGroup(ItemGroup group, NonNullList<ItemStack> list) {
-                new ModifierInstance<>(ItemStack.class, EssenceModifierRegistration.SILK_TOUCH_MODIFIER.get(), 1, null),
-                new ModifierInstance<>(ItemStack.class, EssenceModifierRegistration.ENCHANTED_MODIFIER.get(), 1, null),
-                new ModifierInstance<>(ItemStack.class, EssenceModifierRegistration.FIERY_MODIFIER.get(), 5, null)
+                new ModifierInstance<>(ItemStack.class, () -> EssenceModifierRegistration.EXPANDER_MODIFIER.get(), 2, null),
+                new ModifierInstance<>(ItemStack.class, () -> EssenceModifierRegistration.SILK_TOUCH_MODIFIER.get(), 1, null),
+                new ModifierInstance<>(ItemStack.class, () -> EssenceModifierRegistration.ENCHANTED_MODIFIER.get(), 1, null),
+                new ModifierInstance<>(ItemStack.class, () -> EssenceModifierRegistration.FIERY_MODIFIER.get(), 5, null)
             ));
             if (!list.contains(stack)) {
                 list.add(stack);
             }
             stack = new ItemStack(this, 1, EssenceItemstackModifierHelpers.getStackNBTForFillGroup(
-                new ModifierInstance<>(ItemStack.class, EssenceModifierRegistration.CASCADING_LUMBER_MODIFIER.get(), 1, null),
-                new ModifierInstance<>(ItemStack.class, EssenceModifierRegistration.LUCK_MODIFIER.get(), 5, null),
-                new ModifierInstance<>(ItemStack.class, EssenceModifierRegistration.ENCHANTED_MODIFIER.get(), 1, null),
-                new ModifierInstance<>(ItemStack.class, EssenceModifierRegistration.FIERY_MODIFIER.get(), 5, null)
+                new ModifierInstance<>(ItemStack.class, () -> EssenceModifierRegistration.CASCADING_LUMBER_MODIFIER.get(), 1, null),
+                new ModifierInstance<>(ItemStack.class, () -> EssenceModifierRegistration.LUCK_MODIFIER.get(), 5, null),
+                new ModifierInstance<>(ItemStack.class, () -> EssenceModifierRegistration.ENCHANTED_MODIFIER.get(), 1, null),
+                new ModifierInstance<>(ItemStack.class, () -> EssenceModifierRegistration.FIERY_MODIFIER.get(), 5, null)
             ));
             if (!list.contains(stack)) {
                 list.add(stack);
             }
             stack = new ItemStack(this, 1, EssenceItemstackModifierHelpers.getStackNBTForFillGroup(
-                new ModifierInstance<>(ItemStack.class, EssenceModifierRegistration.CASCADING_LUMBER_MODIFIER.get(), 1, null),
-                new ModifierInstance<>(ItemStack.class, EssenceModifierRegistration.SILK_TOUCH_MODIFIER.get(), 1, null),
-                new ModifierInstance<>(ItemStack.class, EssenceModifierRegistration.ENCHANTED_MODIFIER.get(), 1, null),
-                new ModifierInstance<>(ItemStack.class, EssenceModifierRegistration.FIERY_MODIFIER.get(), 5, null)
+                new ModifierInstance<>(ItemStack.class, () -> EssenceModifierRegistration.CASCADING_LUMBER_MODIFIER.get(), 1, null),
```

Comment:
RegistryObjects are already suppliers. (Hence why this have get, you don't need to create a supplier.